css/components/_lightbox.scss

```diff
@@ -0,0 +1,204 @@
+body.lb-disable-scrolling {
+    overflow: hidden;
+}
+
+.lightbox {
+    position: absolute;
+    left: 0;
+    width: 100%;
+    z-index: 10000;
+    text-align: center;
+    line-height: 0;
+    font-weight: normal;
+    outline: none;
+
+    &Overlay {
```

Comment:
Is this class comes from external library?

Reply:
Yes, it is